frontend/src/static/js/v_authorship.js

```diff
@@ -245,20 +251,24 @@ window.vAuthorship = {
             ? this.getContributionFromAllAuthors(contributionMap)
             : contributionMap[this.info.author];
 
-        if (lineCnt) {
+        if (this.isValidFile(contributionMap, lineCnt, file.isBinary)) {
           const out = {};
           out.path = file.path;
           out.lineCount = lineCnt;
           out.active = lineCnt <= COLLAPSED_VIEW_LINE_COUNT_THRESHOLD;
           out.wasCodeLoaded = lineCnt <= COLLAPSED_VIEW_LINE_COUNT_THRESHOLD;
           out.fileType = file.fileType;
+          out.isBinary = file.isBinary;
```

Comment:
Now all the non-binary file will have this section as undefined, pls make them have false value

Reply:
Sure, no problem